modules/tensor_mechanics/include/postprocessors/CriticalTimeStep.h

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ElementPostprocessor.h"
+#include "GuaranteeConsumer.h"
+#include "ComputeStressBase.h"
+
+// Forward Declarations
+class CriticalTimeStep;
+
+template <>
+InputParameters validParams<CriticalTimeStep>();
+
+class CriticalTimeStep : public ElementPostprocessor, public GuaranteeConsumer
```

Comment:
Can you please add few comments regarding what this class if for as well as what the member variables are for. This level of documentation should be for developers not users of the class, it can be short, but a little something is needed.